File: Python_Assesment_01-02-25/Test3.py
#library management system
class Library:
    def __init__(self):
        self.title = input("Enter the name of the book:")
        self.author = input("Enter the author of the book:")
        self.isbn = input("Enter the ISBN number of the book:")
    
    def display(self):
        print("Title:", self.title)
        print("Author:", self.author)
        print("ISBN:", self.isbn)

book1 = Library()
book1.display()

File: CSharp_Assessment_24-02-2025/test3.cs
using System;
public class Library
{
    public string Title { get; set; }
    public string Author { get; set; }
    public string Isbn { get; set; }
    public Library()
    {
        Title = "Title";
        Author = "Author";
        Isbn = "Isbn";
    }
    public Library(string title, string author)
    {
        Title = title;
        Author = author;
    }

    public Library(string title, string author, string isbn)
    {
        Title = title;
        Author = author;
        Isbn = isbn;
    }

}

public class MainClass
{
    public static void Main()
    {
        Library l = new Library("Title", "Author", "Isbn");
        Library l1 = new Library("Title", "Author");
        Library l2 = new Library();
    }
}